python/lsst/pipe/supertask/configOverrides.py

```diff
@@ -103,3 +120,15 @@ def applyTo(self, config):
                     # this can throw
                     value = eval(value, {})
                     setattr(obj, field[-1], value)
+            elif otype == 'namesDict':
+                try:
+                    parsedNamesDict = ast.literal_eval(override)
+                    if not isinstance(parsedNamesDict, dict):
+                        raise ValueError
```

Comment:
I gather than raising the type rather than an instance works here (because you're about to catch it), but it still kinda weirds me out.  Maybe `ValueError()`?  Meh.  I guess I don't care much.  Maybe you'll have a better alternative than what I could come up with.